be/dbcon.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/*
+ * Creates connection to database
+ *
+ * @param N/A
```

Comment:
if no params can just leave this line out

Reply:
done